### DISCUSSION TITLE
v2.6.0

### DISCUSSION THREAD

**Discussion by sojan-official** [chatwoot/chatwoot#4870] [Releases & Announcements]
Hello everyone, Chatwoot v2.6 is live. Here’s a quick overview. ⬇️

### 🎮 Manage your conversations better with bulk actions
Now you can apply actions to multiple conversations at once, saving you time. Simply select conversations from your dashboard, and assign labels, agents, or change their status. 

### ✏️ Access your WhatsApp templates straight from Chatwoot
WhatsApp doesn't let you send business messages within your chats without pre-approved templates. You can configure these templates through WhatsApp API providers. The good news is that now you can access and use these templates right from your Chatwoot dashboard. Easy, peasy!

### 🔍 More useful downloadable reports
Now you can adjust your "Agents Overview" report for business hours and download it. You will also now be able to download your CSAT reports.

### 🪴 Other updates and improvements 
- Added the APIs for KBase Articles.
- Added the support for human_tag which lets you send Instagram messages outside the 24-hour window.
- Various bug fixes and enhancements. 
- New languages: Ukrainian (UK)

Thanks to @devinellis , @jordan-brough for the contributions

<hr /><em>This discussion was created from the release <a href='https://github.com/chatwoot/chatwoot/releases/tag/v2.6.0'>v2.6.0</a>.</em>

**Comment by digitall-it** [comment]
Congratulations! What does the ability to perform bulk actions on conversations allows to do?

**Reply by sojan-official** [reply]
@digitall-it  you can select multiple conversations at once and perform actions like resolving, assigning to a specific agent, adding labels, etc. 

https://www.loom.com/share/6b55e165a17d498caae956fcd06bb617

**Comment by glauberssoutodossantos** [comment]
Very nice. Could you post a video how " WhatsApp templates straight from Chatwoot" works?

**Reply by proton1k** [reply]
I would like to learn that too! According to the [release process description](https://www.chatwoot.com/docs/handbook/engineering/release-process), it should also be deployed pretty soon to the chatwoot.com, right?

**Reply by sojan-official** [reply]
please refer : https://www.chatwoot.com/blog/v2-6-0/

**Comment by sojan-official** [comment]
## 2.6.0 Dev Changelog

### API Endpoints

#### Additions

- `/api/v1/accounts/:account_id/inboxes/:id/assignable_agents`
    - PR : https://github.com/chatwoot/chatwoot/pull/4722
    - description: Endpoint accepts array of inboxes and return agents with access to it. Used for bulk Actions
- `/api/v1/accounts/:account_id/dashboard_apps` [ CRUD ]
    - PR: https://github.com/chatwoot/chatwoot/pull/4761
    - Description: •Allow users to create and manage dashboard apps to give agents more context
- `/api/v1/accounts/:account_id/portals` [ CRUD ] [ Sub: articles, categories ]
    - PR: https://github.com/chatwoot/chatwoot/pull/4633
    - https://github.com/chatwoot/chatwoot/pull/4777
    - Description: APIs for upcoming Knowledge base feature.
- `/api/v1/accounts/:account_id/csat_survey_responses/download`
    - PR: https://github.com/chatwoot/chatwoot/pull/4694
    - Description: Endpoint that lets you download CSAT Reports in csv format
- `/api/v1/accounts/#{account.id}/conversations` [ Accepts additional parameter :  { reply_status: 'unattended'  } ]
    - PR: https://github.com/chatwoot/chatwoot/pull/4708
    - Description: Ability to filter unattended conversations

#### Deletions

- all routes under `/kbase` have been removed in favor of /portals endpoint

---

### Payload Changes

#### Additions

- Inbox Object
    - `auto_assignment_config`: JSON object returning configuration related to auto assignments. Primarily for enterprise edition
    - `additional_attributes` : For API Channel, this attribute returns the stored templates as a JSON array
    - `message_templates` : For Whatsapp Channel, this attribute returns the stored templates as a JSON array

#### Deletions

- none
